samples/hello_world/src/main.cpp

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2023, Meta
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+
+class Greeter {
+public:
+	static void greet();
+};
+
+void Greeter::greet()
+{
+	printf("Hello C++ World! %s\n", CONFIG_BOARD);
+}
+
+int main(void)
+{
+	Greeter::greet();
+	return 0;
+}
```

Comment:
I'd suggest sticking to the even more minimalistic "official" Hello World https://www.stroustrup.com/hello_world.c

Reply:
`<iostream>` is a beast - it's probably best to avoid for embedded / rtos purposes.

But maybe it works 🤷🏼‍♂️